documentations/guides/creating-a-new-package.md

```diff
@@ -0,0 +1,46 @@
+# Creating a new package
+
+There are many steps involved in creating a new package, and you can contribute at each stage of the process.
+
+## Filing issues
+
+Discovered a problem that you'd like us to solve? Have an idea for a new package?
+Submit a [feature request](https://github.com/Shopify/web-foundation/issues/new?template=FEATURE_REQUEST.md).
+
+## Development
+
+Once we've identified a package you would like to add, it's time to begin the process of creating a PR. If you haven't set up your development environment, there are instructions for getting started in our [contributing guide](../../.github/CONTRIBUTING.md).
+
+You should create your new package with the generator. Simply run `yarn generate:package` and follow the prompts to set up the initial boilerplate.
+
+### Naming
+
+If you need help with finding a good name for your package, feel free to reach our in `#web-foundation-tech` on Slack, or else just pick a relevant name and add a comment to the PR asking for feedback.
```

Comment:
There are a lot of mentions of the slack channel, but this is a public repo. Is this doc meant to be aimed at Shopifolk specifically?

Reply:
I can 🔥 every mention of slack if you prefer? I will also open a PR in Quilt to sync with this repo.